plugins/kotlin/gradle/gradle-java/src/org/jetbrains/kotlin/idea/gradleJava/configuration/KotlinSourceSetDataService.kt

```diff
@@ -285,15 +285,10 @@ class KotlinSourceSetDataService : AbstractProjectDataService<GradleSourceSetDat
                     modelsProvider.findIdeModule(data)?.name
                 }
 
-                if (kotlinSourceSet.isTestModule) {
-                    testOutputPath = (kotlinSourceSet.compilerArguments as? K2JSCompilerArguments)?.outputDir
-                        ?: (kotlinSourceSet.compilerArguments as? K2JSCompilerArguments)?.outputFile
-                    productionOutputPath = null
-                } else {
-                    productionOutputPath = (kotlinSourceSet.compilerArguments as? K2JSCompilerArguments)?.outputDir
-                        ?: (kotlinSourceSet.compilerArguments as? K2JSCompilerArguments)?.outputFile
-                    testOutputPath = null
-                }
+                testOutputPath = (compilerArguments as? K2JSCompilerArguments)?.outputDir
```

Comment:
This is aligned with [facetSerialization](https://github.com/JetBrains/kotlin/blob/8b258426ee4ceaf4b161aa30a2d4a09775dcc2b9/jps/jps-common/src/org/jetbrains/kotlin/config/facetSerialization.kt#L184) implementation since populate the `testOutputPath` and `productionOutputPath` without taking into consideration if facet is test or not